app-shells/bash/files/bash-5.1-parallel_make.patch

```diff
@@ -0,0 +1,15 @@
+bashline.c:65:10: fatal error: builtins/builtext.h: No such file or directory
```

Comment:
This patch does not seem to be applied anywhere. Drop it maybe?

Reply:
Sure, I've dropped the other patches as well which were not needed.